tests/all/test_module.rs

```diff
@@ -316,27 +316,25 @@ fn test_print_to_file() {
 
 #[test]
 fn test_get_set_target() {
-    Target::initialize_x86(&Default::default());
```

Comment:
Why was this removed? This is generally to ensure LLVM's globals are initialized properly

Reply:
The test runs fine without it. Getting/setting the module's target triple does not touch the `TargetRegistry`. It's probably better to put it back in though, as a defensive measure, because I'm not sure that no code path touches the `TargetRegistry` given a module on which you've set the target triple.

Reply:
LLVM's sparse documentation makes it hard for me to tell exactly what preconditions the APIs have